waitress/app/tests/test_routes.py

```diff
@@ -208,6 +215,26 @@ def test_can_regularize_guest_names(self):
         regularized = regularize_guest_names(guest_list)
         assert regularized[-1].firstname == "Guest 10"
 
+    def test_user_can_view_admin_login_page(self):
+        response = self.client.get("")
+        assert response.status_code == 200
+        assert response.templates[0].name == "login.html"
```

Comment:
Use the provided django utility testing method please

Reply:
do you have a link to that? I don't understand what you mean here?

Reply:
I mean the self.assert(s).. provided by the Inherited (TestCase) class

Reply:
Fixed.